diff -ru snowflake-v2.11.0.a/vendor/github.com/pion/dtls/v3/config.go snowflake-v2.11.0.b/vendor/github.com/pion/dtls/v3/config.go
--- snowflake-v2.11.0.a/vendor/github.com/pion/dtls/v3/config.go	2025-03-28 20:24:08.000000000 +0100
+++ snowflake-v2.11.0.b/vendor/github.com/pion/dtls/v3/config.go	2025-03-28 20:32:32.067893022 +0100
@@ -230,7 +230,7 @@
 
 const defaultMTU = 1200 // bytes
 
-var defaultCurves = []elliptic.Curve{elliptic.X25519, elliptic.P256, elliptic.P384} //nolint:gochecknoglobals
+var defaultCurves = []elliptic.Curve{elliptic.P384, elliptic.X25519, elliptic.P256} //nolint:gochecknoglobals
 
 // PSKCallback is called once we have the remote's PSKIdentityHint.
 // If the remote provided none it will be nil
diff -ru snowflake-v2.11.0.a/vendor/github.com/pion/dtls/v3/conn.go snowflake-v2.11.0.b/vendor/github.com/pion/dtls/v3/conn.go
--- snowflake-v2.11.0.a/vendor/github.com/pion/dtls/v3/conn.go	2025-03-28 20:24:08.000000000 +0100
+++ snowflake-v2.11.0.b/vendor/github.com/pion/dtls/v3/conn.go	2025-03-28 20:31:17.697885791 +0100
@@ -31,7 +31,7 @@
 	initialTickerInterval = time.Second
 	cookieLength          = 20
 	sessionLength         = 32
-	defaultNamedCurve     = elliptic.X25519
+	defaultNamedCurve     = elliptic.P256
 	inboundBufferSize     = 8192
 	// Default replay protection window is specified by RFC 6347 Section 4.1.2.6
 	defaultReplayProtectionWindow = 64
